ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.8.yaml

```diff
@@ -0,0 +1,161 @@
+base_images:
+  base:
+    name: "4.9"
```

Comment:
Why is this 4.9?

Reply:
Line 32 gives us cluster with requested version for E2E test.

Reply:
Should be fine. Just an image builder it appears.